ui/lib/kv/addon/components/page/secret/overview.hbs

```diff
@@ -101,6 +103,7 @@
   <KvPathsCard @backend={{@backend}} @path={{@path}} @isCondensed={{true}} />
 </Hds::Card::Container>
 
+{{! @subkeys is null for community edition or if a user does not have read permissions }}
 {{#if @subkeys.subkeys}}
-  <KvSubkeysCard @subkeys={{@subkeys.subkeys}} @canPatchSecret={{@canPatchSecret}} @backend={{@backend}} @path={{@path}} />
+  <KvSubkeysCard @subkeys={{@subkeys.subkeys}} @isPatchAllowed={{@isPatchAllowed}} @backend={{@backend}} @path={{@path}} />
```

Comment:
passing `@isPatchAllowed` is more than we need since we _just_ need patch capabilities here but reusing the arg seemed most sensible. If it makes more sense to folks I can change the conditional to `{{#if @isPatchAllowed }}` though either way we'll have to check for `@subkey.subkeys` in case the version has been deleted